app/commands/mentor/discussion/update_timed_out.rb

```diff
@@ -0,0 +1,39 @@
+class Mentor::Discussion::UpdateTimedOut
+  include Mandate
+
+  def call
+    update_student_timed_out!
+    update_mentor_timed_out!
+  end
+
+  private
+  def update_student_timed_out!
+    student_timed_out_discussions.find_each do |discussion|
+      Mentor::Discussion::StudentTimeOut.(discussion)
+    rescue StandardError => e
+      Bugsnag.notify(e)
+    end
+  end
+
+  def student_timed_out_discussions
+    Mentor::Discussion.includes(:student).
+      awaiting_student.
+      where('awaiting_student_since < ?', timeout_date)
+  end
+
+  def update_mentor_timed_out!
+    mentor_timed_out_discussions.find_each do |discussion|
+      Mentor::Discussion::MentorTimeOut.(discussion)
+    rescue StandardError => e
+      Bugsnag.notify(e)
+    end
+  end
```

Comment:
I have a preference for putting this method above the one above it as its referered to earlier in the file. I like keeping the action methods directly under `private` where possible.